libraries/chem-meta/src/rdkit-api.ts

```diff
@@ -14,6 +14,8 @@ export interface RDModule {
   _malloc(size: number): any;
```

Comment:
Does it really return "any"? I know it's not part of your commit, but please check